src/simulator_core/solver/network/assets/base_asset.py

```diff
@@ -149,7 +148,14 @@ def add_thermal_equations(self, connection_point: int) -> EquationObject:
         :return: An equation object representing the thermal equation.
         :rtype: EquationObject
         """
-        if self.prev_sol[IndexEnum.discharge + connection_point * NUMBER_CORE_QUANTITIES] > 0:
+        if (
```

Comment:
Zou de prev_sol ook niet gerefactored moeten worden? Het is nu nog steeds een omslachtige oplossing. Terwijl je vaak zat moet weten wat van de vorige oplossing op connectie punt x de discharge of andere property is/was?

Reply:
misschien wel,  maar de huidige implementatie vervult de functie en het dat lijkt me niet in deze PR thuishoren

Reply:
Ja,  even (offline) overleggen wat de inhoud van het issue moet zijn